### PR TITLE
Redone without forking

### PR DESCRIPTION
**Pull request by IliaFrolov**

### CODE REVIEW

File: homework/froi/index.js [hunk 1]
```diff
@@ -1 +1,42 @@
-console.log(`Hello World!`);
\ No newline at end of file
+function FizzBuzz () {
+   const numbers = [];
+   for (let i = 1; i <= 100; i++) {
+      let n = i;
```
Comment (kovm) [inline homework/froi/index.js]: this variable is redundant

File: homework/froi/index.js [hunk 2]
```diff
@@ -1 +1,42 @@
...
+      switch (true) {
```
Comment (kovm) [inline homework/froi/index.js]: switch (true) - is a bad design

File: homework/froi/index.js [hunk 3]
```diff
@@ -1 +1,42 @@
...
+      case (n % 3 === 0 && n % 5 === 0):
+         n = 'FizzBuzz';
+         break;
+      case (n % 3 === 0):
+         n = 'Fizz';
+         break;
+      case (n % 5 === 0):
+         n = 'Buzz';
+         break;
+      default:
```
Comment (kovm) [inline homework/froi/index.js]: empty default means _your do it wrong_
rework with another logical operators or make a switch more obvious

File: homework/froi/index.js [hunk 4]
```diff
@@ -1 +1,42 @@
...
+      }
+      numbers.push(n);
+   }
+   console.log(numbers.join(', '));
+   document.body.append(numbers.join(', '));
```
Comment (kovm) [inline homework/froi/index.js]: There is no document, _Neo_.
Just console.log

File: homework/froi/index.js [hunk 5]
```diff
@@ -1 +1,42 @@
...
+}
+FizzBuzz();
+
+function isPolindrome (subject) {
+   if (typeof subject !== 'number' || typeof subject !== 'string' || subject === undefined) {
```
Comment (kovm) [inline homework/froi/index.js]: `if (typeof subject !== 'number' || typeof subject !== 'string')` that's enough in this case

File: homework/froi/index.js [hunk 6]
```diff
@@ -1 +1,42 @@
...
+      return false;
+   }
+   return subject.toLowerCase()
+      .split('').filter(element => element !== ' ')
```
Comment (kovm) [inline homework/froi/index.js]: Take a look at \s
_Matches a white space character, including space, tab, form feed, or line feed._
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions

File: homework/froi/index.js [hunk 7]
```diff
@@ -1 +1,42 @@
...
+      default:
+      }
+      numbers.push(n);
+   }
+   console.log(numbers.join(', '));
+   document.body.append(numbers.join(', '));
+}
+FizzBuzz();
+
+function isPolindrome (subject) {
```
Comment (kovm) [inline homework/froi/index.js]: isPolindrome (131); throws an exception

**Comment by IliaFrolov** [comment]
All have done

**Pull request merged**